packages/ledger/src/lib/ledger.ts

```diff
@@ -219,6 +220,41 @@ const Ledger: WalletBehaviourFactory<HardwareWallet> = async ({
       return getAccounts();
     },
 
+    async verifyOwner({ message = "verify owner", signerId, publicKey } = {}) {
```

Comment:
Currently the NEAR Ledger app can _only sign valid transactions_; it will not be able to sign any payload that is not a transaction -- so this is not implemented currently and should be throwing a 'Method not Supported' type error.